etc/config.yml

```diff
@@ -1,3 +1,16 @@
+selected-storage-adapter: local
```

Comment:
Does this mean that the default adapter has now changed from `s3` to `local`? I'm not sure if there are other considerations here, but if possible could we set this back to `s3` just to avoid having to do a major version change to release this feature (as this is changing default functionality, so it wouldn't be backwards compatible)?

Reply:
Oh for sure, that wouldn't be a problem at all. I wasn't aware S3 was default :)